internal/authorization/converters.go

```diff
@@ -204,3 +204,95 @@ func (c RoleConverter) Map(r *http.Request) []Permission {
 		{Relation: relation(r), ResourceID: fmt.Sprintf("%s:%s", c.TypeName(), role_id)},
 	}
 }
+
+// GET /api/v0/groups
+// GET /api/v0/groups/{id}
+// POST /api/v0/groups
+// PATCH /api/v0/groups/{id}
+// DELETE /api/v0/groups/{id}
+// GET /api/v0/groups/{id}/roles
+// POST /api/v0/groups/{id}/roles
+// DELETE /api/v0/groups/{id}/roles/{r_id}
+// GET /api/v0/groups/{id}/entitlements
+// POST /api/v0/groups/{id}/entitlements
+// DELETE /api/v0/groups/{id}/entitlements/{e_id}
+// GET /api/v0/groups/{id}/identities
+// PATCH /api/v0/groups/{id}/identities
+// DELETE /api/v0/groups/{id}/identities/{i_id}
+type GroupConverter struct{}
+
+func (c GroupConverter) TypeName() string {
+	return GROUP_TYPE
+}
+
+func (c GroupConverter) Map(r *http.Request) []Permission {
```

Comment:
question(possible issue): one thing we don't do here is actually check for the first `id` (`group_id`) to be non-blank, may raise issues at runtime if we don't validate endpoints pathvariables before this middleware is run. Or maybe `chi` does it on its own?

Could it be beneficial to add as a first check after parameter retrieval if the `group_id` is empty and the method allows for an empty parameter, like GET ( -> `can_view`) or POST ( -> `can_edit`)?